### PR TITLE
feature/64193 Request PS Report for specific schools

### PR DESCRIPTION
**Pull request by GuyHarwood**
- Add ability to specify CSV list of URNs when requesting PS report

### CODE REVIEW

File: tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export interface PsReportListSchoolsIncomingMessage {
   requestedBy: string
   dateTimeRequested: string
   jobUuid: string
+  urns?: number[]
```
Comment (jon-shipley) [inline tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts:8]: or undefined?

**Review by jon-shipley — APPROVED** [review]
LGTM 👍

File: tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts (hunk 1 above)
Comment (GuyHarwood) [reply tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts:8]: that's what the question mark is for

Comment (jon-shipley) [reply tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts:8]: I think the question mark means the property may be omitted.  But in the definition in the admin app the property may be there with an undefined value.

Comment (GuyHarwood) [reply tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts:8]: yeah good point, i will update this

**Pull request merged**